### DISCUSSION TITLE
Is it still possible to view what is inside of a queue?

### DISCUSSION THREAD

**Discussion by Parogar** [comfyanonymous/ComfyUI#6040] [General]
I recently updated, and I'm not loving the new queue manager. Previously, we could actually open it up and see what exactly was queued, how many items, and even cancel some individually while leaving others. The new one doesn't seem to allow this.

**Comment by LukeG89** [comment]
I think you can do pretty much everything:
- See all the generation of a single task
- Show the "flat list" to see all the generations
- Delete single tasks
- Load workflows
- Delete all tasks


https://github.com/user-attachments/assets/e872cb2e-3a48-44bd-928f-5a01aaf38c40

There are some feature requests in [ComfyUI_frontend](https://github.com/Comfy-Org/ComfyUI_frontend) to improve other things in the Queue windows
- https://github.com/Comfy-Org/ComfyUI_frontend/issues/1502
- https://github.com/Comfy-Org/ComfyUI_frontend/issues/1422
- https://github.com/Comfy-Org/ComfyUI_frontend/issues/1578
- https://github.com/Comfy-Org/ComfyUI_frontend/issues/1635
- https://github.com/Comfy-Org/ComfyUI_frontend/issues/1823

If you want other improvements, you can open a feature request yourself:
https://github.com/Comfy-Org/ComfyUI_frontend/issues/new/choose

**Comment by BrechtCorbeel** [comment]
What I am wondering is what happened to the manager?

**Reply by LukeG89** [reply]
@BrechtCorbeel The manager is always there. Are you using the Desktop app? I saw other people having this problem today

**Reply by BrechtCorbeel** [reply]
No manager is an add-on, I keep forgetting. I get why they don't integrate on windows it's so slow.
